src/javascript/register.jsx

```diff
@@ -44,4 +44,11 @@ registry.add('callback', 'content-editor', {
     }
 });
 
+// Register GWT Hooks
+window.top.jahiaGwtHook = {
```

Comment:
where will this function be called? have you completed the implementation?

Reply:
this function is called by GWT